src/components/WebexActivityStream/WebexActivityStream.js

```diff
@@ -1,12 +1,44 @@
 import React, {Fragment} from 'react';
 import PropTypes from 'prop-types';
+import {ListSeparator} from '@momentum-ui/react';
+import {format, isToday, isSameWeek, isYesterday} from 'date-fns';
 
 import {RoomType} from '../../adapters/RoomsAdapter';
 import {useActivityStream, useRoom} from '../hooks';
 import WebexActivity from '../WebexActivity/WebexActivity';
 
 import './WebexActivityStream.scss';
 
+/**
+ * Returns a formatted timestamp based on the given date's offset from the current time.
+ *
+ * Divisor for messages from today display today
+ * Divisor for messages from dates from a previous day display as Yesterday
+ * Divisor for messages of dates from a previous day of the week (but not yesterday) display as <day of the week>
+ * Divisor for messages of dates older than a week from today display as M/D/YY
+ *
+ * @param {Date} timestamp Date instance to format
+ * @returns {string} formattedDate
+ */
+export function formatTimeRulerText(timestamp) {
```

Comment:
we do have another function that does the same in `Activity` component. maybe it's time to extract it out and use it in here and there.

Reply:
They are similar but the business logic is different enough to warrant separate functions.

Reply:
gotcha

Reply:
> Right, the time rulers format does not include time.

Ironically. 🤣